src/main/resources/application.properties

```diff
@@ -21,4 +21,15 @@ security.enabled=true
 security.oauth2.resource.userInfoUri=https://www.googleapis.com/userinfo/v2/me
 security.oauth2.resource.filter-order=3
 
-registry.protocols=https
+spring.mail.host=@smtp-host@
+spring.mail.port=587
+spring.mail.test-connection=false
+spring.mail.properties.mail.smtp.auth=false
+spring.mail.properties.mail.smtp.ssl.enable=false
+
+mail.from=amp-dev@ebi.ac.uk
+mail.to=amp-dev@ebi.ac.uk
```

Comment:
These properties should be populated at the time of deployment maybe from environment variables that are not visible. Otherwise we will be food for spammers.